common/build.gradle

```diff
@@ -24,17 +31,13 @@ dependencies {
     implementation project(":public")
     implementation project(":utils")
     implementation project(":strings")
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
-    implementation "androidx.core:core-ktx:$androidxCoreVersion"
-    implementation "com.google.crypto.tink:tink-android:$tinkVersion"
-    implementation "com.android.billingclient:billing:$billingVersion"
+
+    implementation libs.kotlin.stdlib
```

Comment:
We might not need to add the kotlin stdlib... But we can remove it in a different PR if needed.

Reply:
Sorry I was confused, we talked and I think this is needed. In newer projects, the kotlin-dom is added instead, but we can keep this for now.